.github/workflows/test-oblt-cli-cluster-create-ccs.yml

```diff
@@ -0,0 +1,95 @@
+name: test-oblt-cli-cluster-create-ccs
+
+on:
+  merge_group: ~
+  workflow_dispatch:
+    inputs:
+      dry-run:
+        description: 'Dry run'
+        required: true
+        default: false
+        type: boolean
+  push:
+    paths:
+      - '.github/workflows/test-oblt-cli-cluster-create-ccs.yml'
+      - 'oblt-cli/cluster-create-ccs/**'
+
+permissions:
+  contents: read
+
+jobs:
+  test:
+    if: always()
+    needs:
+      - dry-run
+      - wait
+    runs-on: ubuntu-latest
+    steps:
+      - id: check
+        uses: elastic/oblt-actions/check-dependent-jobs@v1
+        with:
+          jobs: ${{ toJSON(needs) }}
+      - run: ${{ steps.check.outputs.is-success }}
+
+  dry-run:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: elastic/oblt-actions/git/setup@v1
+      - name: Get token
+        id: get_token
+        uses: tibdex/github-app-token@3beb63f4bd073e61482598c45c71c1019b59b73a # v2.1.0
+        with:
+          app_id: ${{ secrets.OBS_AUTOMATION_APP_ID }}
+          private_key: ${{ secrets.OBS_AUTOMATION_APP_PEM }}
+          permissions: >-
+            {
+              "contents": "write",
+              "pull_requests": "read"
+            }
+          repositories: >-
+            ["observability-test-environments"]
+      - uses: ./oblt-cli/cluster-create-ccs
+        id: create
+        with:
+          github-token: ${{ steps.get_token.outputs.token }}
+          remote-cluster: 'release-oblt'
+          cluster-name-prefix: 'testgithubaction'
+          gitops: true
+          dry-run: ${{ github.event.inputs.dry-run != '' && github.event.inputs.dry-run || true }}
+      - name: Assert is not empty
+        run: test -n "${{steps.create.outputs.cluster-name}}"
+
+  wait:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: elastic/oblt-actions/git/setup@v1
+      - name: Get token
+        id: get_token
+        uses: tibdex/github-app-token@3beb63f4bd073e61482598c45c71c1019b59b73a # v2.1.0
+        with:
+          app_id: ${{ secrets.OBS_AUTOMATION_APP_ID }}
+          private_key: ${{ secrets.OBS_AUTOMATION_APP_PEM }}
+          permissions: >-
+            {
+              "contents": "write",
+              "pull_requests": "read"
+            }
+          repositories: >-
+            ["observability-test-environments"]
+      - uses: ./oblt-cli/cluster-create-ccs
+        id: create
+        with:
+          github-token: ${{ steps.get_token.outputs.token }}
+          remote-cluster: 'release-oblt'
+          cluster-name-prefix: 'testgithubaction'
+          gitops: true
+          wait: '15'
+      - name: Assert is not empty
+        run: test -n "${{ steps.create.outputs.cluster-name }}"
+      - uses: elastic/oblt-actions/oblt-cli/cluster-destroy@v1
```

Comment:
It could fail because the PR has not been merged yet. We check the CI pass so the cluster is created, but we can only destroy it when the PR is merged. The reason is that waiting for the PR to merge can take 30-60 seconds more, and destroying a cluster in that short time is not usual.

Reply:
I added `continue-on-error: true`. It's not intended to be part of the test-suite, but to tear-down the infra as a best effort rather than having those projects running for 24 hours (those clusters are only created as part of the e2e tests, but they are not needed) 

Should I add a sleep of 60 seconds to play a bit safer?